test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/Search/IncludeSearchTests.cs

```diff
@@ -849,7 +850,39 @@ public async Task GivenARevIncludeIterateSearchExpressionWithSingleIteration_Whe
 
             ValidateBundle(
                 bundle,
-                Fixture.Patient,
+                Fixture.PatiPatient,
+                Fixture.AdamsPatient,
+                Fixture.SmithPatient,
+                Fixture.TrumanPatient,
+                Fixture.AdamsMedicationRequest,
+                Fixture.SmithMedicationRequest,
+                Fixture.AdamsMedicationDispense,
+                Fixture.SmithMedicationDispense);
+
+            ValidateSearchEntryMode(bundle, ResourceType.Patient);
+
+            ValidateBundleUrl(Client.HttpClient.BaseAddress, ResourceType.Patient, query, bundle.Link[0].Url);
+
+            // ensure that the included resources are not counted
+            bundle = await Client.SearchAsync(ResourceType.Patient, $"{query}&_summary=count");
+            Assert.Equal(4, bundle.Total);
+
+            // ensure that the included resources are not counted when _total is specified and the results fit in a single bundle.
+            bundle = await Client.SearchAsync(ResourceType.Patient, $"{query}&_total=accurate");
+            Assert.Equal(4, bundle.Total);
+        }
+
+        [Fact]
```

Comment:
We are missing the [HttpIntegrationFixtureArgumentSets(DataStore.SqlServer)] attribute here since we want the test to run only against Sql

Reply:
Great catch! @namadabu